### ISSUE TITLE
Issues with Neon.1

### ISSUE THREAD

**Issue by homacs** [boothen/Json-Eclipse-Plugin#6]
What I noticed after a quick test on Linux (Ubuntu 16.4):

1. No syntax highlighting on existing json files.
2. Deletes existing line when writing a closing curled bracket.

**Comment by msteiger** [comment]
Thanks for reporting.

Is it somehow tied to Neon.1 on Ubuntu 16 or did it happen in an older release, too? Do you see any errors in the log? Does the syntax highlighting work for any file?

**Comment by homacs** [comment]
Issue 1 might be a false flag, because I did not check whether Eclipse just used the default Text Editor plugin, which I used before on the same files.

To produce the second issue, I created a new Json file via New -> Others -> JSON -> Json File (if I remember correct) and wrote a basic json object, like:
{
    "name": 1
}

But when typing the close bracket, the content just vanished.

It might be worth mentioning that I installed Eclipse from the scratch: I downloaded just Eclipse platform and added only those parts I really need (JDT, CDT, SVN, GIT, TeXlipse, and a few other).

Haven't tried in different setups or investigated any further.

**Comment by msteiger** [comment]
Should be fixed in version 1.0.1 now - can you confirm?

**Comment by pedronavf** [comment]
I also don't get syntax highlighting when opening a file. I'm using Ubuntu 16.04. The Error Log tab shows this:

```
eclipse.buildId=4.6.2.M20161124-1400
java.version=1.8.0_111
java.vendor=Oracle Corporation
BootLoader constants: OS=linux, ARCH=x86_64, WS=gtk, NL=en_US
Framework arguments:  -product org.eclipse.epp.package.cpp.product
Command-line arguments:  -os linux -ws gtk -arch x86_64 -product org.eclipse.epp.package.cpp.product

org.eclipse.ui
Error
Tue Feb 14 13:02:37 PST 2017
Unhandled event loop exception

org.eclipse.swt.SWTException: Failed to execute runnable (java.lang.NullPointerException)
	at org.eclipse.swt.SWT.error(SWT.java:4533)
	at org.eclipse.swt.SWT.error(SWT.java:4448)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:185)
	at org.eclipse.swt.widgets.Display.runAsyncMessages(Display.java:4536)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:4154)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$4.run(PartRenderingEngine.java:1121)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:336)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1022)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:150)
	at org.eclipse.ui.internal.Workbench$5.run(Workbench.java:693)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:336)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:610)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:148)
	at org.eclipse.ui.internal.ide.application.IDEApplication.start(IDEApplication.java:138)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:673)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:610)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1519)
	at org.eclipse.equinox.launcher.Main.main(Main.java:1492)
Caused by: java.lang.NullPointerException
	at com.boothen.jsonedit.editor.JsonTextEditor.updateMarkers(JsonTextEditor.java:324)
	at com.boothen.jsonedit.editor.JsonTextEditor.updateProblemMarkers(JsonTextEditor.java:316)
	at com.boothen.jsonedit.editor.model.JsonReconcilingStrategy$1.run(JsonReconcilingStrategy.java:128)
	at org.eclipse.swt.widgets.RunnableLock.run(RunnableLock.java:35)
	at org.eclipse.swt.widgets.Synchronizer.runAsyncMessages(Synchronizer.java:182)
	... 24 more
```

**Comment by msteiger** [comment]
Should have been fixed with b32d2fc803cee0d18be now. Please re-open if it still exists.
